src/plugins/cosmos/components/DefiModalHeader/DefiModalHeader.tsx

```diff
@@ -4,15 +4,25 @@ import { Text } from 'components/Text'
 
 type DefiModalHeaderProps = {
   headerImageSrc: string
+  headerImageWidth: number
   headerText: string | [string, Record<string, string>]
 }
 
-export const DefiModalHeader = ({ headerImageSrc, headerText }: DefiModalHeaderProps) => (
+export const DefiModalHeader = ({
+  headerImageSrc,
+  headerText,
+  headerImageWidth
```

Comment:
Yeah it is a `maxWidth`, but in effect, since we have a `minWidth`, I like to see it as the actual width, the `minWidth` being a fallback, thus the naming. But you're right, might as well be as declarative as possible.